guides/common/modules/con_prerequisites-for-configuring-project-with-keycloak-authentication.adoc

```diff
@@ -0,0 +1,16 @@
+[id="prerequisites-for-integrating-project-with-keycloak_{context}"]
+= Prerequisites for Configuring {Project} with {Keycloak} Authentication
+
+Before configuring {Project} with {Keycloak} external authentication, ensure that you meet the following requirements:
+
+* A working installation of {Keycloak} server that uses HTTPS instead of HTTP.
```

Comment:
Why we need to mentioned it HTTPS explicitly ? @rabajaj0509  any thoughts ?

Reply:
Explicit is better than implicit :)